cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -88,19 +179,44 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
         query("SHOW USERS", assertNodesShown("User", column = "user")) {
           resultTable()
         }
+        p("It is possible to assign multiple roles to multiple users in one command.")
+        query("GRANT ROLES role1, role2 TO user1,user2,user3", ResultAssertions( r => {
+          assertStats(r, systemUpdates = 4)
+        })) {
+          // Can't pull on the result since neither roles nor users exist
+        }
+      }
+      section("Revoking roles from users", "administration-security-roles-revoke") {
+        p("Users can lose access rights by revoking roles from them using `REVOKE ROLE`.")
+        query("REVOKE ROLE myrole FROM jake", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          p("Nothing is returned from this query, except the count of system database changes made.")
+          resultTable()
+        }
+        p("The roles revoked from users can no longer be seen in the list provided by `SHOW USERS`.")
+        query("SHOW USERS", assertNodesShown("User", column = "user")) {
+          resultTable()
+        }
+        p("It is possible to revoke multiple roles from multiple users in one command.")
+        query("REVOKE ROLES role1, role2 FROM user1,user2,user3", ResultAssertions( r => {
+          assertStats(r, systemUpdates = 4)
+        })) {
+          // Can't pull on the result since neither roles nor users exist
```

Comment:
This comment seems to contradict the `systemUpdates = 4` above?

Reply:
Since the 4 should be a 6 it might be that that is not checked either

Reply:
If I don't print the resultTable() then the statistics are not asserted on, is there a way of doing the asset while not printing the table to the docs? (I have several places with empty bodies or just printing a paragraph that does not actually test the assertion)